tools/ci/docker-containers-windows

```diff
@@ -25,10 +25,11 @@ export AGENTCTL_IMAGE=grafana/agentctl
 if [ -n "$DRONE_TAG" ]; then
   VERSION=$DRONE_TAG
 else
+  git config --global --add safe.directory C:/drone/src/
```

Comment:
This should probably be something done at the Drone level rather than in the script IMO (also what does it do?)